### PR TITLE
Ignore non-rc prereleases

### PR DESCRIPTION
**Pull request by mathias-baumann-sociomantic**

### CODE REVIEW

**Comment by leandro-lucarella-sociomantic** [comment]
Better, but now it's trying to release v1.0.0-rc1 instead of v1.8.0-rc1:
```
tangort:v1.x.x= ✔ $ neptune-release -p
Please enter the upstream remote name.
Will default to [origin]: 
We are on branch v1.x.x
Detected release v1.0.0-rc1
Warning: No corresponding milestone found for v1.0.0-rc1
Actions to be done:
 * Creating annotated tag v1.0.0-rc1 (["git", "tag", "-F-", "v1.0.0-rc1", "v1.x.x"])
 * Checkout original branch v1.x.x (["git", "checkout", "v1.x.x"])

Would you like to continue? y/n: 
```

File: src/release/main.d [hunk 1]
```diff
@@ -726,14 +726,23 @@ ActionList prepareMajorRelease ( ref HTTPConnection con, ref Repository repo,
 
     auto current_branch = SemVerBranch(getCurrentBranch());
 
+    static bool branchExists ( SemVerBranch branch )
+    {
+        import std.process : executeShell;
```
Comment (leandro-lucarella-sociomantic) [inline src/release/main.d]: Please use `execute()` instead of `executeShell()` now! (`execute(["git", "rev-parse", "--verify", branch])`)

Comment (mathias-baumann-sociomantic) [reply src/release/main.d]: You can't tell me what to do! You're not even my real mum!

**Comment by mathias-baumann-sociomantic** [comment]
* Added test
* Added fixes for some other issues

**Comment by mathias-baumann-sociomantic** [comment]
ping @leandro-lucarella-sociomantic

**Comment by leandro-lucarella-sociomantic** [comment]
I did the reviewing in https://github.com/sociomantic-tsunami/neptune/pull/128. I prefer to merge everything there.

**Pull request closed without merging**